request.example

```diff
@@ -19,40 +19,25 @@
 			},
 			"modifiers": [
 				{
-					"stat": {
-						"advance": 0,
-						"march": 0,
-						"discipline": 0,
-						"health_point": 0,
-						"defense": 0,
-						"resilience": 0,
-						"armour": 0,
-						"aegis": 0,
-						"attack": 0,
-						"offensive": 0,
-						"strength": 0,
-						"armour_penetration": 0,
-						"agility": 0
-					},
-					"bonus": false,
-					"nb_dice": 0,
-					"requirements": []
+					"Global": {
+                        "advance": 3,
+                        "march": 2,
+                        "discipline": 1
+                    }
```

Comment:
Because that's how the modifier are in the game's rule They can be global, offensive, defensive or weapons(close and long range). Depending on the type, only a few fields are impacted, not all the stats. So it's easier for the back to just send the fields impacted by the modifier.
Furthermore, it's better for us too as it's how we represents our modifier